server/controllers/availability.js

```diff
@@ -4,25 +4,24 @@ const { google } = require("googleapis");
 
 // @desc FreeBusy
 // @route GET /freebusy?date=143213434&meetingTypeID=df34234fgdg235
-availabilitiesRouter.get("/freebusy", (request, response) => {
-    // request.query.meetingTypeID
-    // request.query.date // must be in datetime format
+availabilitiesRouter.get("/freebusy", async (request, response) => {
+    const { day, month, year } = request.query;
```

Comment:
```suggestion
    const { day, month, year, duration, user } = request.query;
```

Reply:
I believe this has to be 
```suggestion
    let{ day, month, year, duration, user } = request.query;
```
in order to reassignment of
```js
duration = Number(duration)
user = await User.findbyId(user)
```
in the suggestions below: